apps/api/src/domain/models/user-management/group/entities/invariant-validation/coscradUserGroupInvariantValidation.spec.ts

```diff
@@ -129,33 +27,11 @@ describe('CoscradUserGroup.validateInvariants', () => {
 
     describe('when the data is invalid', () => {
         describe('when a simple invariant (generalized type) rule fails', () => {
-            Object.entries(dataSchema).forEach(([propertyName, propertySchema]) => {
-                describe(`when the property: ${propertyName} is invalid`, () => {
-                    const invalidValuesToUse = generateInvalidValuesForProperty(propertySchema);
-
-                    invalidValuesToUse.forEach(({ value, description }) => {
-                        describe(`when given the invalid value: ${JSON.stringify(
-                            value
-                        )} (${description})`, () => {
-                            it('should return the expected error', () => {
-                                const invalidDto = buildInvalidGroupDto({
-                                    id: value,
-                                });
-
-                                const invalidInstance = new CoscradUserGroup(invalidDto);
-
-                                const result = invalidInstance.validateInvariants();
-
-                                assertCoscradDataTypeError(
-                                    result as InternalError,
-                                    'id',
-                                    InvalidCoscradUserGroupDTOError
-                                );
-                            });
-                        });
-                    });
-                });
-            });
+            assertTypeErrorsFromInvalidFuzz(
+                CoscradUserGroup,
+                new CoscradUserGroup(validGroupDto),
```

Comment:
maybe we should just pass the DTO since the helper can new it up using the Ctor that is already passed.

Reply:
done